libs/blocks/marketo/marketo-multi.js

```diff
@@ -0,0 +1,96 @@
+import { createTag } from '../../utils/utils.js';
+
+const VALIDATION_STEP = {
+  name: '2',
+  phone: '2',
+  mktoFormsJobTitle: '2',
+  mktoFormsFunctionalArea: '2',
+  company: '3',
+  state: '3',
+  postcode: '3',
+  mktoFormsPrimaryProductInterest: '3',
+};
```

Comment:
Will we eventually want this configuration to be set up elsewhere or by authors?

Reply:
There are multiple directions on how this would be configured, but right now this feature is more of a test.
We're not sure what the authoring would look like and it may by configured by the template definitions.